doc/developer-guides/c_coding_guidelines.rst

```diff
@@ -3083,11 +3086,12 @@ Compliant example::
        uint32_t* showcase_func(void);
 
 
-C-CS-10: A single space shall exist around binary and ternary operators
-=======================================================================
+C-CS-10: Single spaces shall exist around binary and ternary operators
```

Comment:
The plural ``Single spaces`` seems odd here.  I'd vote to leave this as it was written

Reply:
Restored the original wording.